recipes/armadillo/all/test_v1_package/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.6)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(armadillo REQUIRED)
+
+add_executable(example example.cpp)
```

```suggestion
add_executable(example ../test_package/example.cpp)
```
You can re-use it from test_package, so you don't need to copy example.cpp again.